### PR TITLE
[Vmware] Use host FQDN if host IP is not found on VNC command

### PR DESCRIPTION
**Pull request by nvazquez**
### Description

This PR uses the host name if the host IP is not found on Vmware for Vmware 7 console proxy websocket URL 
Fixes: #5413 

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

**Comment by nvazquez** [comment]
@blueorangutan package

**Comment by nvazquez** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
@blueorangutan test centos7 vmware-67u3

**Comment by rhtyd** [comment]
@blueorangutan test centos7 vmware-70

File: vmware-base/src/main/java/com/cloud/hypervisor/vmware/mo/VirtualMachineMO.java [hunk 1]
```diff
@@ -1177,14 +1178,23 @@ public boolean configureVm(Ternary<VirtualDevice, VirtualDeviceConfigSpecOperati
         return false;
     }
 
+    private String getHostFqdn(HostMO hostMO, String hostName) {
```
Comment (sureshanaparti) [inline vmware-base/src/main/java/com/cloud/hypervisor/vmware/mo/VirtualMachineMO.java:1181]: @nvazquez I think, it is better to keep this method in HostMO.

```
public String getHostFqdn() {
        try {
             HostDnsConfig hostDnsConfig = getHostNetworkInfo().getDnsConfig();
             return String.format("%s.%s", hostDnsConfig.getHostName(), hostDnsConfig.getDomainName());
         } catch (Exception e) {
             s_logger.error("Could not get host FQDN for host: " + getHostName());
         }
         return null;
}
```

**Comment by nvazquez** [comment]
I'm closing this in favour of #5419

**Pull request closed without merging**